forta-uniswap-py/README.md

```diff
@@ -0,0 +1,29 @@
+# Large Uniswap Detector Agent
+
+## Description
+
+This agent detects large uniswap transactions
+
+## Supported Chains
+
+- Ethereum
+
+## Alerts
+
+Describe each of the type of alerts fired by this agent
+
+- AE-UNISWAP
```

Comment:
Probably not a huge deal for this example, but if we ever do more Uniswap agents we might want more specific alert names. Something like `AE-UNISWAP-TXN` or something

Reply:
Sure, do you think `TXN` or `LARGE-SWAP` would be better

Reply:
Let's go `LARGE-SWAP`

Reply:
We are going to use `AE-UNISWAP-LARGESWAP-METHODID` since another large swap agent will be written but will filter on log events